### DISCUSSION TITLE
Welcome to Rentalx Discussions!

### DISCUSSION THREAD

**Discussion by ermessonlima** [ermessonlima/Rentalx#3] [Q&A] [answered]
### Discussed in https://github.com/ermessonlima/Rentalx/discussions/1

<div type='discussions-op-text'>

<sup>Originally posted by **ermessonlima** June 10, 2022</sup>

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

</div>

**Comment by ermessonlima** [comment]
Hello!

**Reply by oliveiragabriel1** [reply]
Hello

**Reply by oliveiragabriel1** [reply]
> Hello!

Send me pix!!!

**Comment by LelioH** [comment]
I've got do this for the big brain achievement

**Reply by oliveiragabriel1** [reply]
Me to bro

**Reply by ermessonlima** [reply]
Send pix!

**Reply by oliveiragabriel1** [reply]
> I've got do this for the big brain achievement

Send me too

**Comment by oliveiragabriel1** [comment, marked as answer]
I need help! Can you send me pix, please?

**Reply by LelioH** [reply]
On my next payment
